kong/router/atc.lua

```diff
@@ -685,9 +497,9 @@ function _M:exec(ctx)
     local req_scheme = ctx and ctx.scheme or var.scheme
 
     local err
-    match_t, err = self:select(req_method, req_uri, req_host, req_scheme,
+    match_t, err = self:select(CACHE_PARAMS.method, req_uri, req_host, req_scheme,
```

Comment:
Do you still need to pass these? It seems that `CACHE_PARAMS` already contains it and setting it again inside `select`.

Reply:
done.